content/documentation/Ox64/Software/Flashing.adoc

```diff
@@ -257,8 +257,9 @@ First, write the normal Linux image to the SD card if you have not done so alrea
  cd ~/ox64/buildroot/output/images # for built images
  sudo dd if=/sdcard.img of=/dev/sdb bs=1M conv=fsync status=progress
 
-Run the following command to re-read the partition tables. Re-inserting the SD card works too.
+Run the following commands to re-read the partition tables. Re-inserting the SD card works too.
 
+ sudo umount /media/*/*
```

Comment:
Ehhh you'd only be running the re-read partitions command immediately after `dd`ing the SD card, so nothing would be mounted.

(If new/old partitions were mounted during the the `dd` operation, you have bigger problems..)

Reply:
ok, can you fix my error? I'm a bit lost with all the revision I'm doing. sorry